### PR TITLE
remove deprecated dropshadows

### LINKED ISSUE #725: Remove deprecated dropshadows
🌱 **Are you new to the codebase? Welcome! Please see [the contributing guidelines](https://github.com/learningequality/kolibri-design-system/blob/develop/CONTRIBUTING.md).**

## Blocked by

- https://github.com/learningequality/kolibri/issues/12552
- https://github.com/learningequality/kolibri-design-system/issues/724

## Summary

For both internal KDS components styling and external use by non-KDS components in our products, according to the updated guidelines [in the "Elevation and shadow > Drop shadows" documentation section](https://deploy-preview-723--kolibri-design-system.netlify.app/styling/#elevation), we only need to keep `%dropshadow-1dp` , `%dropshadow-2dp` , and `%dropshadow-6dp` in the *definitions.scss*:

[https://github.com/learningequality/kolibri-design-system/blob/0cbdf57d24690cb6c23618ba700118a831cd9ffc/lib/styles/definitions.scss#L59-L157](https://github.com/learningequality/kolibri-design-system/blob/0cbdf57d24690cb6c23618ba700118a831cd9ffc/lib/styles/definitions.scss#L59-L157)

All other `%dropshadow-...` should not be available for use.

## Guidance

- I think it’d be fine to just comment them out rather than delete, in case we need some of them in the future - as long as there’s a note that uncommenting needs to be discussed with designers and should be aligned with the guidelines. 
- Issues in "Blocked by" should be resolved first to ensure that KDS version upgrade is not demanding in Kolibri that has many drop shadows. Still needs to be communicated as a breaking change and  consumers should double-check before installing the corresponding release (the issues mentioned should cover most of it, however there may be one or two places in Studio to update as well)

### PR DESCRIPTION
**Pull request by lokesh-sagi125**
## Description
remove deprecated dropshadows

#### Issue addressed
 https://github.com/learningequality/kolibri-design-system/issues/725

Addresses #725 

## Changelog

  - **Description:** removed the deprecated dropshadows
  - **Products impact:**  none
  - **Addresses:** #725 
  - **Components:** -
  - **Breaking:** no
  - **Impacts a11y:** no
  - **Guidance:**  -

### CODE REVIEW

**Review by ozer550 — APPROVED** [review]

**Review by ozer550 — DISMISSED** [review]
I just saw that in the issue description it is mentioned rather to comment these out and not remove them completely.

**Comment by lokesh-sagi125** [comment]
oh , sorry about that @ozer550  i got confused and thought its the other way around 😅, just reverted the delete and commented the deprecated dropshadows:).

**Review by AlexVelezLl — CHANGES_REQUESTED** [review]
Thank you @lokesh-sagi125! These are the right changes for this, however, for now, we are checking if these 3 values ​​1dp, 2dp and 6dp also aligned correctly to Studio's needs, so I will block the merge of this PR for now until we have a final confirmation from our design team :hugs: thank you!

File: lib/styles/definitions.scss [hunk 1]
```diff
@@ -67,91 +67,91 @@ $core-time: 0.15s;
...
-}
-%dropshadow-4dp {
-  box-shadow: 0 2px 4px -1px rgba(0, 0, 0, 0.2), 0 4px 5px 0 rgba(0, 0, 0, 0.14),
-    0 1px 10px 0 rgba(0, 0, 0, 0.12);
-}
-%dropshadow-5dp {
-  box-shadow: 0 3px 5px -1px rgba(0, 0, 0, 0.2), 0 5px 8px 0 rgba(0, 0, 0, 0.14),
-    0 1px 14px 0 rgba(0, 0, 0, 0.12);
-}
+// %dropshadow-3dp {
```
Comment (AlexVelezLl) [inline lib/styles/definitions.scss:72]: Could we add here a comment indicating that uncommenting needs to be discussed with designers and should be aligned with the guidelines.

**Review by AlexVelezLl — COMMENTED** [review]
Alright, it seems that these will work fine with Studio :tada:. Lets just add this comment that was mentioned in the issue,and after that, this should be good to go!

File: .gitignore [hunk 2]
```diff
@@ -22,3 +22,4 @@ docs/jsdocs.js
 .idea
 kolibri-design-system.iml
 docs/pages/playground.vue
+package.json
```
Comment (AlexVelezLl) [inline .gitignore]: Lets remove this line, please, we need to track the changes of the package.json

File: package.json [hunk 3]
```diff
@@ -83,5 +83,8 @@
   },
   "browserslist": [
     "extends browserslist-config-kolibri"
-  ]
+  ],
+  "volta": {
```
Comment (AlexVelezLl) [inline package.json]: In KDS we still use node 10. Please, lets remove this addition

**Review by AlexVelezLl — CHANGES_REQUESTED** [review]
Hey @lokesh-sagi125! Lets remove these extra additions, and after that we can proceed to merge the PR. Thank you!

**Comment by lokesh-sagi125** [comment]
@AlexVelezLl  just removed the additions

**Review by AlexVelezLl — APPROVED** [review]
Thank you @lokesh-sagi125! This looks good to me, merging :tada:.

**Pull request merged**